proprietary/tokens/src/components/amsterdam/ordered-list.tokens.json

```diff
@@ -0,0 +1,26 @@
+{
+  "amsterdam": {
+    "ordered-list": {
+      "color": { "value": "{amsterdam.color.primary-black}" },
+      "font-family": { "value": "{amsterdam.typography.font-family}" },
+      "font-size": {
+        "narrow": { "value": "{amsterdam.typography.text-level.6.font-size.narrow}" },
+        "wide": { "value": "{amsterdam.typography.text-level.6.font-size.wide}" }
+      },
```

Comment:
For design token naming, it works when in the first iteration you do `font-size` first and after that `narrow` or `wide`.

What's tricky is introducing modifiers at a later stage, if you want to have backwards compatibility.

For example, you can't introduce `font-weight.narrow` and `font-weight.wide` while keeping support for `font-weight` without modifiers. Because of `"font-size": { "value" .... }` other properties such as `"narrow": { }` will not be recognized as design tokens.

In other NL Design System repositories we use another naming order, for this reason.

Reply:
Ah, so the convention is that CSS properties should always be the deepest level, and modifiers should be defined higher up? So in this case, instead of `font-size.narrow` and `font-size.wide`, you would have `narrow.font-size` and `wide.font-size`?

Makes sense. Is this convention documented anywhere, as far as you know?

And would you suggest the same convention for the brand level design tokens? So `amsterdam.typography.text-level.6.narrow.font-size`?

Reply:
Yes, the new token names you propose are what I would suggest.

Convention is documented here:
https://nldesignsystem.nl/meedoen/design-tokens/

Related Style Dictionary issue is here:
https://github.com/amzn/style-dictionary/issues/366

For brand level concepts I would probably group various aspects of the same concept in the same manner, yes. `font-size` and `line-height` are typically a pair, for example.

The design token names can get annoyingly long this way (`font-weight-scale.semi-bold.font-weight` or something like that), so I must confess sometimes I take arbitrary decisions, but usually I regret them later on 😂

Reply:
Thanks! I'll check all design tokens to make sure they follow this convention in a seperate PR, I'll resolve this conversation for now